dev-tex/pythontex/pythontex-0.14.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_2,3_3} )
+
+inherit latex-package python-single-r1
+
+DESCRIPTION="Fast Access to Python from within LaTeX"
+HOMEPAGE="https://github.com/gpoore/pythontex"
+SRC_URI="https://github.com/gpoore/${PN}/archive/v${PV}.zip"
```

Comment:
So deal with it.

Look for "Customization of Output File Names in SRC_URI" in
http://devmanual.gentoo.org/ebuild-writing/eapi/index.html